### ISSUE TITLE
Problem with mammal_generic.lua behaviour since latest update

### ISSUE THREAD

**Issue by Rickaitch** [TheGameCreators/GameGuruRepo#3639]
Hi all, I'm new to GGM but not to programming and system design etc. Love this program but I'm stumped with a change in my project which I can't explain... I have 6 Rabbits 1 Fox and several Rats running around in this game, and where they used to all take off in different directions etc when I'd approach, they now just sit there running through the preening (idle) animation. I did at one point change the behaviour of one of the Rats to some of the others available but changed back to the original.

Now it doesn't matter which mammal I place on the level it just sits there idling. The default mammal_generic.lua has not been tampered with or edited. Is this a problem introduced in the latest update? (I can't see how), or has some other form of corruption to the default behaviour occurred?

24 hours later:

OK, I'm pretty sure this has to be a problem introduced with the last update... I've done clean GGM installs on 3 other machines and deliberately NOT tried my project on them. Instead, I started a new project with a new level and placed the Player and several of the Mammals. Running the level, as with my project, the Mammals behaviour is to just sit preening. They don't run away when the player approaches (as before). This happens on all 3 of the clean installs.

Can someone please test this? And is there a bug notification system or do the developers check these posts? Thanks for any help!

24 hours later (again)

The Trigger zone seems to somehow be involved...  done some more testing on one of the clean installs, and using the TZ seems to somehow corrupt the operation of an attached .lua script.  Script errored out once after I'd associated a fox with the TZ and then went to trigger it's appearance in the scene.

**Comment by davetgc** [comment]
Confirmed.

**Comment by ZakJudges** [comment]
Hi @Rickaitch, thanks for reporting this issue. A new Experimental build is available on Steam with the Fix.

@smallg2020 Indeed you are correct, IntersectStaticPerformant() was returning nil, due to missing the ignorePlayerCapsule parameter. I have added the extra parameter to the IntersectStaticPerformant() call, so that it will not return nil. I have also added a check for obj being nil, like your fix above. Thanks :)
